### ISSUE TITLE
[typescript] OverlayView doesn't have all props declared in type definition file

### ISSUE THREAD

**Issue by chrfrasco** [tomchentw/react-google-maps#738]
Not sure if this is the correct way to specify the problem that I'm having, but I am getting several type errors with this code (copy pasted from the docs):

```javascript
<OverlayView
  position={theGridLatLng}
  mapPaneName={OverlayView.OVERLAY_MOUSE_TARGET}
  getPixelPositionOffset={getPixelPositionOffset}
>
  ...
</OverlayView>
```

- `[ts] Property 'position' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<OverlayView> & Readonly<{ children?: ReactNode; }>...'.`
- `[ts] Property 'OVERLAY_MOUSE_TARGET' does not exist on type 'typeof OverlayView'.`
- `[ts] Property 'getPixelPositionOffset' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<MyOverlayView> & { children?: ReactNode; position?...'.`

None of these properties are declared in the type definition file:

```javascript
export interface OverlayViewProps {
    mapPaneName?: string
    children?: ReactNode
}

export default class OverlayView extends Component<OverlayViewProps> {
    getPanes(): google.maps.MapPanes
    getProjection(): google.maps.MapCanvasProjection
}
```

This is the way I'm fixing it currently:

```javascript
class MyOverlayView extends OverlayView {
  static OVERLAY_MOUSE_TARGET = `overlayMouseTarget`
  props: {
    children?: React.ReactNode
    position?: google.maps.LatLng | google.maps.LatLngLiteral
    getPixelPositionOffset?: (w: number, h: number) => { x: number; y: number }
  } & OverlayViewProps
}
```

Are these properties intentionally being left off? If not, I will create a pull request with these properties added

(P.S. Thanks for the great library! You've saved me a lot of time)

**Comment by tomchentw** [comment]
Please just fix the definition files of TypeScript. PRs are welcomed.

**Comment by tomchentw** [comment]
Released [v9.4.5](https://github.com/tomchentw/react-google-maps/releases/tag/v9.4.5)
